### PR TITLE
Use get_all_rows

### PR DESCRIPTION
**Pull request by kaapstorm**
This PR supersedes  #11747. It involved quite a bit of cutting and pasting, but I think it's turned out better in the end.

@emord @esoergel cc @nickpell

### CODE REVIEW

**Comment by emord** [comment]
Looks fine to me.  `stream_es-query` looks nice. I don't think it would work with our ORM though :/

**Comment by esoergel** [comment]
There's a `query.scroll()` method @NoahCarnahan wrote that should fit that use-case though.

**Comment by emord** [comment]
:tight:

File: custom/openclinica/reports.py [hunk 1]
```diff
@@ -4,55 +4,35 @@
 from corehq.apps.reports.generic import GenericReportView
 from corehq.apps.reports.standard import CustomProjectReport
 from corehq.apps.reports.standard.cases.basic import CaseListMixin
+from corehq.elastic import stream_es_query, SIZE_LIMIT
```
Comment (dimagimon) [inline custom/openclinica/reports.py]: F401 'stream_es_query' imported but unused

**Pull request merged**